tool/jt.rb

```diff
@@ -404,8 +404,9 @@ def help
           parser                                     build the parser
           options                                    build the options
           cexts                                      build only the C extensions (part of "jt build")
-          native [--no-sulong] [--no-jvmci] [--no-sforceimports] [extra mx image options]
-                                                     build a native image of TruffleRuby (--no-jvmci to use the system Java)
+          native [--no-sulong] [--no-jvmci] [--no-sforceimports] [--tools] [extra mx image options]
```

Comment:
Should be `--no-tools` since we want a complete image by default, right?

Reply:
ok great. I'll negate it.